hadoop-hdds/docs/content/concept/OzonePorts.md

```diff
@@ -0,0 +1,96 @@
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+
+This document provides a comprehensive overview of the network ports utilized by Apache Ozone. Due to its distributed nature and the requirement for high performance in handling data-intensive tasks, understanding and correctly configuring these network ports is essential for the successful deployment, operation, and maintenance of Apache Ozone clusters.
+
+# **Ozone Manager (OM)**
+
+| Default Port Number | Configuration Key                           | Endpoint Protocol | Purpose |
+|:--------------------|:--------------------------------------------| :---- | :---- |
+| 9862                | ozone.om.address.\<service_id\>.\<node_id\> | Hadooop RPC | Primary RPC endpoint for Ozone clients. |
+| 9874                | ozone.om.http-address.\<service_id\>.\<node_id\>                       | HTTP | Web UI for monitoring OM status and metadata. |
+| 9875                | ozone.om.https-address.\<service_id\>.\<node_id\>                      | HTTPS | Secure Web UI for monitoring OM status and metadata. |
+| 9872                | ozone.om.ratis.port.\<service_id\>.\<node_id\>                         | HTTP/2 | RPC endpoint for OM HA instances to form a RAFT consensus ring |
+| 8981                | ozone.om.grpc.port                          | HTTP/2 | gRPC endpoint for Ozone Manager clients |
+
+# **Storage Container Manager (SCM)**
+
+| Default Port Number | Configuration Key | Endpoint Protocol | Purpose |
+| :---- | :---- | :---- | :---- |
+| 9860 | ozone.scm.client.port | Hadoop RPC | Communication with Ozone clients for namespace and container management. If ozone.scm.client.address is defined (default is empty), ozone.scm.client.address overrides it. |
+| 9863 | ozone.scm.block.client.port | Hadoop RPC | Communication with Datanodes for block-level operations. |
+| 9876 | ozone.scm.http-address | HTTP | Web UI for monitoring SCM status. |
+| 9877 | ozone.scm.https-address | HTTPS | Secure Web UI for monitoring SCM status. |
+| 9861 | ozone.scm.datanode.port | Hadoop RPC | Port used by DataNodes to communicate with the SCM. If ozone.scm.datanode.address is defined (default is empty), ozone.scm.datanode.address overrides it. |
+| 9961 | ozone.scm.security.service.port  | Hadoop RPC | SCM security server port |
```

Comment:
Is it the case that SCM confs also has suffixes?

Reply:
Yes correct -- added an extra line to explain the suffixes.